hw-model/src/lib.rs

```diff
@@ -889,19 +889,22 @@ pub trait HwModel {
         let response = response.ok_or(ModelError::UploadMeasurementResponseError)?;
 
         // Get response as a response header struct
-        let response = mailbox_api::MailboxRespHeader::read_from(response.as_slice())
-            .ok_or(ModelError::UploadMeasurementResponseError)?;
+
+        let response: &StashMeasurementResp =
```

Comment:
From https://ip-v8.github.io/rust-ipv8/zerocopy/struct.LayoutVerified.html#method.new

> new verifies that bytes.len() == size_of::<T>() and that bytes is aligned to align_of::<T>(), and constructs a new LayoutVerified. If either of these checks fail, it returns None.

AFAICT, there is no guarantee that Vec<u8> has 4-byte alignment. Probably best to do:

```
let response = StashMeasurementResp::read_from(response.as_slice())
            .ok_or(ModelError::UploadMeasurementResponseError)?;
```